README.md

```diff
@@ -52,30 +53,51 @@ comes with everything you need to start your research, including:
 
 * Educational examples that work out of the box: MNIST, LSTM seq2seq, Graph Neural Networks, Sequence Tagging
 
-* HOWTO guides -- diffs that add functionality to educational base exampless
+* [HOWTO guides](https://flax.readthedocs.io/en/latest/howtos.html) -- diffs that add functionality to educational base exampless
 
 * Fast, tuned large-scale end-to-end examples: CIFAR10, ResNet ImageNet, Transformer LM1b
 
-### An annotated MNIST example
+## Try Flax now by forking one of our starter examples
 
-See [docs/annotated_mnist.md](docs/annotated_mnist.md) for an MNIST
-example with detailed annotations for each code block.
+### Image Classification
+⟶ [MNIST](examples/mnist) (also see [annotated version](https://flax.readthedocs.io/en/latest/annotated_mnist.html))
 
-### Flax Modules
+⟶ [CIFAR-10](examples/cifar) (Wide ResNet w/ and w/o Shake-Shake, PyramidNet w/ShakeDrop)
+
+⟶ [ResNet50 on ImageNet](examples/imagenet)
+
+### Transformer Models
+⟶ [Sequence tagging on Universal Dependencies](examples/nlp_seq)
+
+⟶ [LM1b language modeling](examples/lm1b) **([try on a TPU in Colab](https://colab.research.google.com/github/google/flax/blob/master/examples/lm1b/Colab_Language_Model.ipynb))**
+
+⟶ (work-in-progress) [WMT translation](https://github.com/google/flax/pull/61)
```

Comment:
The WMT PR changed a ton, so there's a new one at https://github.com/google/flax/pull/133